Art-of-software-testing-guide.md

```
@@ -0,0 +1,340 @@
+# 『The Art of Software Testing』から学ぶソフトウェアテスト実践ガイド
+### 初学者のためのステップバイステップ・ベストプラクティス
+
+> 参考書籍: Glenford J. Myers, Tom Badgett, Corey Sandler『The Art of Software Testing, 3rd Edition』
+> https://www.oreilly.com/library/view/the-art-of/9780471469124/
+
+---
+
+<a id="toc"></a>
+## 目次
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)